src/App.jsx

```diff
@@ -19,9 +19,22 @@ export function App() {
 	 * to create and join a new list.
 	 */
 	const [listToken, setListToken] = useStateWithStorage(
-		'my test list',
+		null,
 		'tcl-shopping-list-token',
 	);
+	/**
+	 * Callback function gets passed as a prop through Home component to retrieve generated token.
+	 * It is then set to localStorage through setListToken and added to the database.
+	 **/
+	const setList = async (token) => {
+		try {
+			setListToken(token);
+			const docToken = await addDoc(collection(db, `${token}`), {});
+			console.log('Document written with ID: ', docToken.id);
```

Comment:
let's use more descriptive logging corresponding to the domain rather than the firebase terms
ie. `New list created with id: ____`

Reply:
I agree and it is an easy fix that merits a quick refactor.